src/elements/content-picker/itemSelectionHelper.js

```diff
@@ -0,0 +1,15 @@
+/**
+ * @flow
+ * @file Functions to determine whether a given item is selected.
+ * @author Box
+ */
+
+export function containsItem(item: BoxItem) {
+    const { id, type } = item;
+
+    return (selected: BoxItem) => selected.id === id && selected.type === type;
+}
+
+export function isSelected(item: BoxItem, selectedItems: Array<BoxItem>) {
```

Comment:
For both of these functions probably should add a return type, so `: boolean` here and `: (item: BoxItem) => boolean` above